plugins/circleci/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "module": "esnext",
```

Comment:
don't change this

Reply:
React.lazy's `import(...)` syntax otherwise isn't accessible, what can be a workaround here?

Reply:
It should work, it's being used in the graphiql plugin.